src/app/dialog/dialog.scss

```diff
@@ -0,0 +1,33 @@
+.dialog {
```

Comment:
could you extract style-only components into folder named `app/styles/` . e.g `app/styles/dialog.scss`